src/modules/verifiableCredentials/verifiableCredentials.types.ts

```diff
@@ -48,3 +68,40 @@ export interface VerifyVerifiableCredentialResults {
   warnings: string[];
   errors: string[];
 }
+
+// Verifiable Presentation Types
+
+export interface SignVerifiablePresentationOptions {
```

Comment:
I think it's better to use `ProofOptions`. It's a bit more succinct and aligns with VC terminology and DIDKit terminology https://github.com/spruceid/didkit/blob/305481916434f97183c783106aff6f3f0056486c/lib/web/src/lib.rs#L134 
```suggestion
export interface ProofOptions {
```